export/export.go

```diff
@@ -107,28 +107,46 @@ func NewExportService(client clientset.Interface, pods PodService, nodes NodeSer
 	}
 }
 
+type options struct {
+	ignoreErr bool
+}
+
+type ignoreErrOption bool
+
+type Option interface {
+	apply(*options)
+}
+
+func (i ignoreErrOption) apply(opts *options) {
+	opts.ignoreErr = bool(i)
+}
+
+func (s *Service) WithIgnoreErr() Option {
```

Comment:
nits: Rename this func to just `IgnoreErr`. And add the doc for this like:

```suggestion
// IgnoreErr is the option to literally ignore errors.
// If it is enabled, the method won't return any errors, but just log errors as error logs.
func (s *Service) IgnoreErr() Option {
```